projects/i18n/interfaces/language.ts

```diff
@@ -62,7 +62,7 @@ export interface TuiLanguageKit {
     /**
      * [@string 'choose day', @param 'choose range', @param 'choose days']
      */
-    mobileCalendarTexts: [string, string, string];
+    mobileCalendarTexts: [string, string, string] | null;
```

Comment:
Could we just use empty array to hide these labels ?

Reply:
`[string, string, string] | []` ?